_example/echo.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"net/http"
+	"github.com/labstack/echo/v4"
+	"github.com/labstack/echo/v4/middleware"
+	"github.com/arl/statsviz"
+)
+
+func main(){
+	// Echo instance
+	e := echo.New()
+
+	// Middleware
+	e.Use(middleware.Logger())
+	e.Use(middleware.Recover())
+
+	// Routes
+	e.GET("/", hello)
```

Comment:
Is is necessary to register that handler? If not I'd prefer to only show the bare minimum

Reply:
Not necessary. I copied from echo repo

Reply:
In that case I'd prefer if you remove it in order to reduce the code to a minimum working example.